### PR TITLE
Consolidate atomtable elements

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
Here we are removing dependence on atomtable.h and replacing it with dependence on elements.h

### CODE REVIEW

File: src/libcsg/modules/io/pdbreader.cc [hunk 1]
```diff
@@ -17,428 +17,460 @@
...
-    _fl.open(file.c_str());
-    if(!_fl.is_open())
-        throw std::ios_base::failure("Error on open topology file: " + file);
+  _fl.open(file.c_str());
+  if (!_fl.is_open())
+    throw std::ios_base::failure("Error on open topology file: " + file);
 
-    NextFrame(top);
-    _fl.close();
+  cerr << "No error on opening" << endl;
```
Comment (junghans) [inline src/libcsg/modules/io/pdbreader.cc]: Why did you add that line?

File: src/libcsg/modules/io/pdbreader.h [hunk 2]
```diff
@@ -23,7 +23,8 @@
 #include <fstream>
 #include <votca/csg/topologyreader.h>
 #include <votca/csg/trajectoryreader.h>
-#include <votca/csg/atomtable.h>
+#include <votca/tools/elements.h>
+//#include <votca/csg/atomtable.h>
```
Comment (junghans) [inline src/libcsg/modules/io/pdbreader.h]: Remove!

**Review by junghans — CHANGES_REQUESTED** [review]
Looking at the whitespace-ignorant diff: https://github.com/votca/csg/pull/242/files?w=1

I saw a few thing!

**Comment by JoshuaSBrown** [comment]
Wait let me check some things. I thought I had already addressed these issues, give me a moment.

**Comment by JoshuaSBrown** [comment]
Ok should be better now.

**Review by junghans — APPROVED** [review]

File: src/libcsg/modules/io/pdbreader.cc [hunk 3]
```diff
@@ -17,428 +17,460 @@
...
+        // elem_sym =  string(line,(77-1),2);
+        // str       , Charge on the atom
+        charge = string(line, (79 - 1), 2);
+      }
+      catch (std::out_of_range &err) {
+        string err_msg = "Misformated pdb file in atom line # " +
+                         boost::lexical_cast<string>(bead_count) +
+                         "\n the correct pdb file format requires 80 "
+                         "characters in width. Furthermore, " +
+                         "\n to read the topology in from a .pdb file the "
```
Comment (jkrajniak) [inline src/libcsg/modules/io/pdbreader.cc:219]: `to read the topology from a ` ?

File: src/libcsg/modules/io/pdbreader.cc [hunk 4]
```diff
@@ -17,428 +17,460 @@
...
+        // Splicing will remove atoms from the now obsolete molecule and place
+        // them on the chosen molecule.
+        molecule_atms[chosen_mol].splice(molecule_atms[chosen_mol].end(),
+                                         molecule_atms[obsolete_mol]);
+
+        // Finally we will clear out the record of the obsolete molecule
+        molecule_atms.erase(obsolete_mol);
+      }
+    }
+#ifdef DEBUG
```
Comment (jkrajniak) [inline src/libcsg/modules/io/pdbreader.cc]: Maybe instead of `DEBUG`, it is better to use 
```#ifndef NDEBUG```
@junghans, do you have any preference?

**Review by jkrajniak — APPROVED** [review]
minor, LGTM

File: src/libcsg/modules/io/pdbreader.cc (hunk 3 above)
Comment (JoshuaSBrown) [reply src/libcsg/modules/io/pdbreader.cc:219]: pdb file. It is a molecular data file.

File: src/libcsg/modules/io/pdbreader.cc (hunk 4 above)
Comment (junghans) [reply src/libcsg/modules/io/pdbreader.cc]: Right, I guess that would be more cmake standard.

Comment (JoshuaSBrown) [reply src/libcsg/modules/io/pdbreader.cc]: Ok will do.
